invoicer_project/static/js/login_request.js

```diff
@@ -0,0 +1,105 @@
+const host = "http://127.0.0.1:8000";
+const emailForm = document.getElementById("email_input_log_in_page");
```

Comment:
I think we should name it emailInput instead of emailForm, to avoid confusing

Reply:
done